[2025-02-02T20:08:08.792+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-02T20:08:08.848+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_processing.nds-load scheduled__2025-02-02T20:06:00+00:00 [queued]>
[2025-02-02T20:08:08.873+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_processing.nds-load scheduled__2025-02-02T20:06:00+00:00 [queued]>
[2025-02-02T20:08:08.874+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-02T20:08:08.921+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): nds-load> on 2025-02-02 20:06:00+00:00
[2025-02-02T20:08:08.941+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1550) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-02T20:08:08.944+0000] {standard_task_runner.py:72} INFO - Started process 1555 to run task
[2025-02-02T20:08:08.946+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_processing', 'nds-load', 'scheduled__2025-02-02T20:06:00+00:00', '--job-id', '706', '--raw', '--subdir', 'DAGS_FOLDER/DAG_etl.py', '--cfg-path', '/tmp/tmps4k_2yms']
[2025-02-02T20:08:08.952+0000] {standard_task_runner.py:105} INFO - Job 706: Subtask nds-load
[2025-02-02T20:08:09.092+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_processing.nds-load scheduled__2025-02-02T20:06:00+00:00 [running]> on host a664bcf281bf
[2025-02-02T20:08:09.810+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_processing' AIRFLOW_CTX_TASK_ID='nds-load' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T20:06:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-02T20:06:00+00:00'
[2025-02-02T20:08:09.813+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-02T20:08:09.893+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:09.981+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:10.206+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:10.371+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:10.505+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:10.652+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:10.816+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:10.966+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:11.174+0000] {base.py:84} INFO - Retrieving connection 'DB_connect'
[2025-02-02T20:08:11.426+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-02T20:08:11.455+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-02T20:08:11.456+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_processing, task_id=nds-load, run_id=scheduled__2025-02-02T20:06:00+00:00, execution_date=20250202T200600, start_date=20250202T200808, end_date=20250202T200811
[2025-02-02T20:08:11.528+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-02T20:08:11.620+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-02T20:08:11.624+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
